README.md

```diff
@@ -78,6 +78,7 @@ to be triggered on `GoyoEnter` and `GoyoLeave` events.
 ```vim
 function! s:goyo_enter()
   silent !tmux set status off
+  silent !tmux resize-pane -Z
```

Comment:
Worked like a charm, didn't think about that case! I have amended your suggestion.